examples/polyglot-zipkin/python-pyramid/README.adoc

```diff
@@ -0,0 +1,16 @@
+= Example application of Python with pyramid_zipkin and swagger_zipkin
+
+Python application instrumented with pyramid_zipkin and swagger_zipkin.
+
+== Standalone Run & Build
+[source,shell]
+----
+$ go install github.com/pavolloffay/hello
```

Comment:
Not sure we should point to your repo - is there any other way to run this standalone from the example in apm?